app/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
 
 dependencies {
 
-    def composeBom = platform('androidx.compose:compose-bom:2022.12.00')
+    def composeBom = platform('androidx.compose:compose-bom:2023.01.00')
     implementation composeBom
     androidTestImplementation composeBom
```

Comment:
Compose it's not used in instrumentation tests and it should be removed

Reply:
Sure, I can handle it in next PR